src/electron.ts

```diff
@@ -36,23 +36,19 @@ app.on('ready', () => {
   win.setMenu(m);
 
   ipcMain.on('minimize', () => {
-    if (!current.isMinimized()) {
-      current.minimize();
-    }
+    current.minimize();
+  });
+
+  ipcMain.on('tabBaximize', () => {
+    current.isMaximized() ? current.unmaximize() : current.maximize();
   });
 
   ipcMain.on('maximize', () => {
-    let isFullScreen = null;
-    if (osPlatform === "linux" || osPlatform === "win32") {
-      isFullScreen = current.isMaximized();
-      if (!isFullScreen) {
-        current.maximize();
-      } else {
-       current.unmaximize()
-      }
+    let notMac = osPlatform === "linux" || osPlatform === "win32";
+    if (notMac) {
```

Comment:
Not sure if this really works, but try to replace the `if` block with this one liner `notMac && current.isMaximized() ? current.unmaximize() : current.maximize();`